library/core/src/task/poll.rs

```diff
@@ -5,6 +5,10 @@ use crate::ops::{self, ControlFlow};
 
 /// Indicates whether a value is available or if the current task has been
 /// scheduled to receive a wakeup instead.
+///
+/// This is the return type of <code>[Future::poll]</code>.
```

Comment:
Is there a reason this can't use rustdoc's item links? https://doc.rust-lang.org/rustdoc/write-documentation/linking-to-items-by-name.html Also a minor rewording suggestion

```suggestion
/// This is the returned by [`Future::poll`](core::future::Future::poll)
```

I notice the `<code>...</code>` links are another place in this file, that should probably be updated too if there isn't a reason.